docs/dev-docs.md

```diff
@@ -0,0 +1,86 @@
+# Development docs
+
+This page should provide basic technical information about a Konveyor Forklift project helpful for engineering onboarding.
+
+## Used technologies
+
+The Konveyor Forklift project is an application based on the Kubernetes/OpenShift operator and controller framework.
+
+### Operator
+Forklift Operator uses ansible and its main task is to install the Forklift into the Kubernetes or Openshift cluster.
+
+Repo: https://github.com/konveyor/forklift-operator
+
+### Controller
+Forklift Controller is written in golang. Its task is to run/orchestate/drive all operations related to the VM migrations. An entrypoint is a reconcile loop which watches events from the cluster and handles received events.
```

Comment:
Please add a note about the (2) separate containers and add a description of the inventory cache.  1. The inventory.  2. The main controller.